components/form/builder/src/Input/index.js

```diff
@@ -94,6 +94,7 @@ const Input = ({
     ...(input.disabled && {
       disabled: true
     }),
+    ...(input.inputmode && {inputMode: input.inputmode}),
```

Comment:
👀 is `inputmode` on `input` object the right property name as it's coming? or is it `inputMode`?

Reply:
It will be defined at JSON as `inputmode` but passed to children components as `inputMode`.